Marlin/src/inc/SanityCheck.h

```diff
@@ -1628,6 +1628,13 @@ static_assert(hbm[Z_AXIS] >= 0, "HOMING_BUMP_MM.Z must be greater than or equal
   #error "POWER_MONITOR_CURRENT_PIN and POWER_MONITOR_VOLTAGE_PIN must be different."
 #endif
 
+/**
+ * Stepper Driver Anti-Reverse Protection
+ */
+#if ENABLED(DRIVER_SAFE_POWER_PROTECT) &&  !PIN_EXISTS(SAFE_POWER)
```

Comment:
I don't think `PIN_EXISTS(SAFE_POWER)` is ever evaluated here. I can comment out the `SAFE_POWER_PIN` and no error is thrown.

Reply:
Yes, nothing will be enabled without `SAFE_POWER_PIN` definition, so this is redundant, and I deleted it in https://github.com/thisiskeithb/Marlin/pull/6